dockstore-webservice/src/main/java/io/openapi/api/impl/ServiceInfoApiServiceImpl.java

```diff
@@ -0,0 +1,93 @@
+package io.openapi.api.impl;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.time.ZoneId;
+import java.util.Date;
+import java.util.Optional;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.SecurityContext;
+
+import io.dockstore.webservice.core.User;
+import io.openapi.api.ServiceInfoApiService;
+import io.openapi.model.Service;
+import io.openapi.model.ServiceType;
+import io.swagger.api.impl.ToolsApiServiceImpl;
+
+import static io.dockstore.common.PipHelper.DEV_SEM_VER;
+
+public class ServiceInfoApiServiceImpl extends ServiceInfoApiService {
+    public static final Date UPDATE_DATE = new Date();
```

Comment:
So update date will be when this class is instantiated, probably when the webservice comes up. 
As in other comment, this is ok for now but worth checking what they meant this date to be.

Reply:
https://github.com/ga4gh-discovery/ga4gh-service-info/issues/65